packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -47,13 +57,13 @@ describe('--schema from project directory', () => {
     const output = stripAnsi(replaceEngineType(result))
     expect(output).toMatchInlineSnapshot(`
 
-      ✔ Generated Prisma Client (0.0.0 | TEST_ENGINE_TYPE) to ./@prisma/client in XXXms
-      You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
-      \`\`\`
-      import { PrismaClient } from './@prisma/client'
-      const prisma = new PrismaClient()
-      \`\`\`
-    `)
+                                          ✔ Generated Prisma Client (0.0.0 | TEST_ENGINE_TYPE) to ./@prisma/client in XXXms
+                                          You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
+                                          \`\`\`
+                                          import { PrismaClient } from './@prisma/client'
+                                          const prisma = new PrismaClient()
+                                          \`\`\`
+                            `)
```

Comment:
Can you undo these changes? Looks like it's still a problem with inline snapshots 😢